codice/alliance
# CAL-299 Add static analysis tools to alliance pipeline

#### What does this PR do?
Add static analysis tools to alliance pipeline
Update coverage numbers in catalog-nsili-endpoint so it will pass on windows builds
Update owasp and surefire versions
Marked VedeoTest.testStreamingVideo as unstable.

#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?
@mcalcote @oconnormi 
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@lessarderic
@shaundmorris

#### How should this be tested?
Verify build works as expected in test environment
Note: Owasp and nsp runs will fail due to findings. The findings will need to be fixed in a separate ticket.
#### Any background context you want to provide?
#### What are the relevant tickets?

[CAL-299](https://codice.atlassian.net/browse/CAL-299)
#### Screenshots (if appropriate)
#### Checklist:
